projects/plugins/jetpack/extensions/shared/components/audio-player/index.jsx

```diff
@@ -74,7 +75,14 @@ function AudioPlayer( {
 
 	useEffect( () => {
 		const audio = audioRef.current;
-		const mediaElement = new MediaElementPlayer( audio, meJsSettings );
+		const mediaElement = new MediaElementPlayer( audio, {
+			...meJsSettings,
+			success: function() {
+				if ( loadWhenReay && audio?.load ) {
+					audio.load();
+				}
+			}
```

Comment:
Perhaps this could be:
```suggestion
			success: loadWhenReady && audio?.load
```